cmd/gator/validate/validate.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+*/
+package validate
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/gator"
+	"github.com/open-policy-agent/gatekeeper/pkg/gator/validate"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```

Comment:
Use yaml.v3. v2 has several parsing bugs that are known but they won't fix.

Reply:
Switched